sections/featured-product.liquid

```diff
@@ -229,6 +229,17 @@
               <div {{ block.shopify_attributes }}>
                 {%- if product != blank -%}
                   <product-form class="product-form">
+                    <div class="product-form__error-message-wrapper" role="alert" hidden>
+                      <span class="visually-hidden">{{ 'accessibility.error' | t }} </span>
```

Comment:
Outputting "Error" isn't required. VO includes a _chime_ when the message is announced. Windows screen readers include, _"Alert"_. Safe to remove this.